code/modules/atmospherics/machinery/airalarm.dm

```diff
@@ -719,6 +731,7 @@
 
 	var/new_area_danger_level = 0
 	for(var/obj/machinery/airalarm/AA in A)
+		AA.manual_alarm = FALSE
```

Comment:
during real atmos alert like plasma in the air, the `manual_alarm` will return FALSE for when air is cleared the air alarm could actually return normal state